test/assertion/get_tgt_val/test_assert_get_tgt_val.py

```diff
@@ -0,0 +1,163 @@
+import unittest
```

Comment:
These 2 test files do not seem to be running on github actions... 👉  https://github.com/QuantestPy/quantestpy/actions/runs/3773774763/jobs/6415475086

Reply:
Please add `__init__.py` in the test dir; ref https://github.com/QuantestPy/quantestpy/pull/168